### DISCUSSION TITLE
Detect change in mutifile yaml

### DISCUSSION THREAD

**Discussion by srijan-shukla** [tj-actions/changed-files#2288] [Q&A] [answered]
I have below yaml 

abc.yaml
---
name: "ElastiCache CPU Utilization"
threshold: 75
period: 60
evaluation_periods: 1
query: "Maximum"
statistic: "Average"
metric: "CPUUtilization"
dimensions:
  CacheClusterId: "test-elasticache"
namespace: "AWS/ElastiCache"
alarm_actions:
  - "arn:aws:sns:ap-south-1:1211212:test-aac-test"
ok_actions: []  # No actions to take when the alarm is OK
comparison_operator: "LessThanThreshold"
datapoints_to_alarm: 1
description: "Alarm when CPU Utilization is greater than 90% for 5 minutes"
---
name: "ElastiCache CPU Utilization"
threshold: 80
period: 60
evaluation_periods: 1
query: "Maximum"
statistic: "Average"
metric: "CPUUtilization"
dimensions:
  CacheClusterId: "agcbwid-test"
namespace: "AWS/ElastiCache"
alarm_actions:
  - "arn:aws:sns:ap-south-1:1112123:test-aac-devops"
ok_actions: []  # No actions to take when the alarm is OK
comparison_operator: "LessThanThreshold"
datapoints_to_alarm: 1
description: "Alarm when CPU Utilization is greater than 90% for 5 minutes"

Basically in a single ...multiple yaml blocks separated by "---" .

I want to fetch only that block in which the change has been made .....not the whole yaml.

How can I do that?

**Comment by jackton1** [comment, marked as answer]
HI @srijan-shukla, This action solely returns the filenames of changed files or supports writing deleted files to a specified location for contents that changed in the file itself. You would need to write a custom script to determine the file content changes.
